src/views/payment-sheet-views/card-view.js

```diff
@@ -263,13 +307,36 @@ CardView.prototype._validateForm = function (showFieldErrors) {
     }
   }
 
-  if (!this._validateCardholderName()) {
-    isValid = false;
+  if (this.extraInputs) {
+    for (i = 0; i < this.extraInputs.length; i++) {
```

Comment:
This could just as easily be a forEach, right? Just use `return` in place of `continue`